yt_dlp/extractor/vrt.py

```diff
@@ -178,154 +178,224 @@ def _real_extract(self, url):
+            'series': 'Taboe',
+            'thumbnail': 'https://images.vrt.be/orig/2025/02/19/8198496c-d1ae-4bca-9a48-761cf3ea3ff2.jpg',
+            'timestamp': 1740286800,
+            'title': 'Mensen met het syndroom van Gilles de la Tourette',
+            'upload_date': '20250223',
+            '_old_archive_ids': ['canvas pbs-pub-f50faa3a-1778-46b6-9117-4ba85f197703$vid-547507fe-1c8b-4394-b361-21e627cbd0fd'],
         },
-        'params': {'skip_download': 'm3u8'},
     }]
     _NETRC_MACHINE = 'vrtnu'
-    _authenticated = False
+
+    _VIDEO_PAGE_QUERY = '''
+    query VideoPage($pageId: ID!) {
+        page(id: $pageId) {
+            ... on EpisodePage {
+                episode {
+                    ageRaw
+                    description
+                    durationRaw
+                    episodeNumberRaw
+                    id
+                    name
+                    onTimeRaw
+                    program {
+                        title
+                    }
+                    season {
+                        id
+                        titleRaw
+                    }
+                    title
+                }
+                ldjson
+                player {
+                    image {
+                        templateUrl
+                    }
+                    modes {
+                        streamId
+                    }
+                }
+            }
+        }
+    }
+    '''
+
+    def _fetch_refresh_token(self):
+        refresh_token = self._get_refresh_token_from_cookie()
+        if refresh_token and not self._is_jwt_token_expired(refresh_token):
+            return refresh_token
+
+        if not self._get_login_info()[0]:
+            return
+
+        refresh_token = self.cache.load(self._NETRC_MACHINE, 'refresh_token', default=None)
+        if refresh_token and not self._is_jwt_token_expired(refresh_token):
+            self.write_debug('Restored refresh token from cache')
+            self._set_cookie('.www.vrt.be', 'vrtnu-site_profile_rt', refresh_token, path='/vrtmax/sso')
+            return refresh_token
+
+    def _fetch_video_token(self):
+        video_token = self._get_video_token_from_cookie()
+        if video_token and not self._is_jwt_token_expired(video_token):
+            return video_token
+
+        if self._get_login_info()[0]:
+            video_token = self.cache.load(self._NETRC_MACHINE, 'video_token', default=None)
+            if video_token and not self._is_jwt_token_expired(video_token):
+                self.write_debug('Restored video token from cache')
+                self._set_cookie('.www.vrt.be', 'vrtnu-site_profile_vt', video_token)
+                return video_token
+
+        refresh_token = self._fetch_refresh_token()
+        if not refresh_token:
+            return None
+
+        self._download_webpage(
+            'https://www.vrt.be/vrtmax/sso/login', None,
+            'Refreshing video token', query={'scope': 'openid,mid'})
```

Comment:
I noticed this doesn't actually work. Should be
```suggestion
        self._download_json(
            'https://www.vrt.be/vrtmax/sso/refresh', None,
            note='Refreshing video token', errnote='Failed to refresh video token')
```